apiserver/facades/client/cloud/cloud_test.go

```diff
@@ -1366,3 +1366,7 @@ func (m *mockModelBackend) AllMachines() ([]credentialcommon.Machine, error) {
 func (m *mockModelBackend) CloudCredential(tag names.CloudCredentialTag) (state.Credential, error) {
 	return state.Credential{}, nil
 }
+
+func (m *mockModelBackend) ControllerConfig() (credentialcommon.ControllerConfig, error) {
+	return nil, nil
```

Comment:
I assume this is needed to satisfy an interface?
If it is not called, it needs to return an NotImplemented error to catch any times we might accidentally call it.

Reply:
sure, it returns NotImplementedf now.